.github/workflows/build.yml

```diff
@@ -84,6 +84,29 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
       - run: yarn install --frozen-lockfile
       - run: yarn test:e2e
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
+    needs: [tests, checks, e2e-tests]
```

Comment:
Could you please also fix the `canary-release` step so it contains the `e2e-tests` prerequisite as well?
I notice it because of this line, the bug was not introduced by your changes though.

Reply:
Makes sense, added` e2e-tests` to `canary-release` step